core/arch/arm/mm/core_mmu.c

```diff
@@ -271,6 +276,62 @@ static struct tee_mmap_region *find_map_by_pa(unsigned long pa)
 	return NULL;
 }
 
+#if defined(CFG_SECURE_DATA_PATH)
+static bool dtb_get_sdp_region(void)
+{
+	void *fdt = NULL;
+	int node = -1;
+	int parent = -1;
+	paddr_t tmp_addr = 0;
+	size_t tmp_size = 0;
+
+	if (!IS_ENABLED(CFG_EMBED_DTB))
+		return false;
+
+	fdt = get_embedded_dt();
+	if (!fdt)
+		panic("No DTB found");
+
+	parent = fdt_path_offset(fdt, "/reserved-memory");
+
+	if (parent < 0)
+		return false;
+
+	fdt_for_each_subnode(node, fdt, parent) {
+		const char *name = NULL;
+
+		name = fdt_get_name(fdt, node, NULL);
+
+		if (strncmp(name, tz_sdp_match, strlen(tz_sdp_match)))
```

Comment:
My understanding is that matching against a DT node name should be avoided if possible. Should we introduce a "compatible" property to indicate the purpose of this reservation (or some other similar mechanism)?